### ISSUE TITLE
Vehicles disappear when subjected to too much physics

### ISSUE THREAD

**Issue by Creepopolous** [StarQuestMinecraft/LimitlessBugs#17]
So I built a birch sled to go down the tallest mountain I saw, and I pressed the I key to push it forward. It disappeared soon after. It wasn't invisible, it disappeared entirely. I built a few more birch vehicles in differing scenarios: down slopes, up slopes, in a different world, on relatively flat ground, down a high ledge. In almost all instances, the vehicle disappeared. The only time it didn't immediately vanish was when it was traveling on the flat platform, with few forces acting on it. It then disappeared after it fell off the ledge. I don't know if this happens with vehicles of other patterns (gravel doesn't get detected), but I wouldn't be surprised if it does.

In short, physics acting on a vehicle for about a second makes it go away.

**Comment by brickworm2012** [comment]
Version of limitless, and windows, mac, or linux?

**Comment by Creepopolous** [comment]
Windows. I chose 64 bit.

**Comment by GingerWalnut** [comment]
This should be fixed for the next release
